config/env/all.js

```diff
@@ -1,9 +1,9 @@
 var path = require('path'),
 rootPath = path.normalize(__dirname + '/../..');
 var keys = rootPath + '/keys.txt';
-
+var localDb = 'mongodb://localhost:27017/cfh';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-    db: process.env.MONGOHQ_URL
+    db: process.env.MONGOHQ_URL || localDb
```

Comment:
@andela-efortune @kolafas you might need to pair on this, so you configure `process` as a blobal in you JSHint file